platform/cavium/docker-syncd-cavm/Dockerfile.template

```diff
@@ -0,0 +1,18 @@
+FROM docker-base
+
+RUN apt-get update
+
+COPY deps /deps
+
+RUN apt-get -y install libpcap-dev libxml2-dev python-dev swig libsensors4-dev redis-tools
```

Comment:
1. We need redis-tools in syncd container to synchronize with redis DB
2. We need dev packages for diagnostics shell support